.gitignore

```diff
@@ -40,6 +40,7 @@ manifest.appcache
 
 /b2g
 /firefox
+/profile-sms/
```

Comment:
__nit:__ you can name you profile folder ```profile-debug``` so that you don't need to update global ```.gitignore``` with sms specific things :)